src/Translations/Visitor/FormOptionLabelsVisitor.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\PhpCs\Translations\Visitor;
+
+use Becklyn\PhpCs\Translations\Visitor\FormOptionsVisitor\ChildFormLabelsVisitor;
+use PhpParser\Node;
+use PhpParser\NodeTraverser;
+use Symfony\Component\Form\AbstractType;
+
+/**
+ *
+ */
+class FormOptionLabelsVisitor extends AbstractVisitor
+{
+    /**
+     * @inheritDoc
+     */
+    public function enterNode (Node $node)
+    {
+        if (!$node instanceof Node\Stmt\Class_)
+        {
+            return;
+        }
+
+        $className = $this->getClassName($node);
+
+        if ("" === $className)
+        {
+            return;
+        }
+
+        // only act on abstract types
+        if (!\is_a($className, AbstractType::class, true))
+        {
+            return;
+        }
+
+
+        $buildFormMethod = $node->getMethod("buildForm");
+
+        if (null === $buildFormMethod)
+        {
+            return;
+        }
+
+        // traverse through form and fetch all options
+        $traverser = new NodeTraverser();
+        $formOptionsVisitor = new ChildFormLabelsVisitor();
+        $traverser->addVisitor($formOptionsVisitor);
+        $traverser->traverse([$buildFormMethod]);
+
+        foreach ($formOptionsVisitor->getMessages() as $message)
```

Comment:
Are we also handling labels that have been added using `FormEvents::*`?

```php
$builder
    ->addEventListener(FormEvents::POST_SET_DATA, function (FormEvent $event) : void
	{
		/** @var SomeClass $someClass */
		$someClass = $event->getData();
		$form = $event->getForm();

		$form
			->add("something", null, [
				"label" => "some_form.something.label",
			]);
	}
```

Reply:
Yes, see below: we are looking for `->add()` on either `$builder` and `$form` variables, and in the event listener we normally use `$form`.